README.md

```diff
@@ -8,6 +8,11 @@ This component provides a tcp server which awaits a message on port 1234 with a
 
 In the configuration, the EEPROM Read feature can be enabled and disabled. If the feature is enabled, the tcp server sends the content of a external EEPROM to the connected client directly after the connection was accepted and before the client shall send the URL. The EEPROM is connected via I2C with the ESP32(S2). The I2C connection can be configured in the configuration menu.
 
+## Task Watchdog
+The ota update server is monitored by a task watchdog, which has to be triggered within a certain time. The allowed time window can be adjusted via the setting CONFIG_ESP_TASK_WDT_TIMEOUT_S in the configuration menu. If the ota update server is in standby and waits for a connecting client, it triggers the watchdog every 3 seconds.
```

Comment:
What happens if the ota server crashes after the diagnostic cb has returned true? Then, the module is just restarted, but no roll-back. Correct?

Reply:
Yes, like every other critical error at this point.